orchestra/management/commands/migrate_todos_status.py

```diff
@@ -0,0 +1,76 @@
+from django.core.management.base import BaseCommand
```

Comment:
I assume @jumasheff is reviewing, so I'll just skim.

One q: on approx ~1M to-dos, how quickly does this script run?

Reply:
4-5 minutes.